### PR TITLE
fix: improve link to chats

### PR DESCRIPTION
**Pull request by TomaszWaszczyk**
I wanted to join Element and Discord chats and saw that the links does not work.

Small fix.

### CODE REVIEW

**Comment by cmichi** [comment]
@TomaszWaszczyk Thanks for the PR, but for me both links work? What are the errors you get? Do both not work for you?

**Comment by TomaszWaszczyk** [comment]
Look at left bottom corner:
[ink.zip](https://github.com/paritytech/ink/files/5506037/ink.zip)

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@
 [Guided Tutorial for Beginners](https://substrate.dev/substrate-contracts-workshop/#/0/building-your-contract) •
 [`cargo-contract`](https://github.com/paritytech/cargo-contract) cli tool for ink! contracts •
 [Canvas UI](https://paritytech.github.io/canvas-ui/#/upload) for contract deployment/interaction •
-Talk to us on [Matrix]([k3]) or [Discord]([l1])
+Talk to us on [Element](https://app.element.io/#/room/!tYUCYdSvSYPMjWNDDD:matrix.parity.io) or [Discord](https://discord.gg/ztCASQE)
```
Comment (cmichi) [inline README.md:37]:
```suggestion
Talk to us on [Element]([k2]) or [Discord]([l2])
```

The variables for those links already exist in the readme.

**Review by cmichi — CHANGES_REQUESTED** [review]
Ah now I got you, thanks for the Gif! The chats are linked two times in the Readme, once as images in the badges and once in the text. I was only looking at the badges.

**Review by cmichi — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@
 [Guided Tutorial for Beginners](https://substrate.dev/substrate-contracts-workshop/#/0/building-your-contract) •
 [`cargo-contract`](https://github.com/paritytech/cargo-contract) cli tool for ink! contracts •
 [Canvas UI](https://paritytech.github.io/canvas-ui/#/upload) for contract deployment/interaction •
-Talk to us on [Matrix]([k3]) or [Discord]([l1])
+Talk to us on [Element]([k2]) or [Discord]([l2])
```
Comment (cmichi) [inline README.md:37]:
```suggestion
Talk to us on [Element][k2] or [Discord][l2]
```

**Pull request merged**